code/game/jobs/job/religions.dm

```diff
@@ -0,0 +1,379 @@
+	var/male_adept = "Chaplain"
+	var/female_adept = "Chaplain"
+	var/bible_type = /obj/item/weapon/storage/bible
+
+	var/list/bible_names = list()
+	var/list/deity_names = list()
+
+	var/list/keys = list() // What you need to type to get this particular relgion.
+
+/datum/religion/New() // For religions with several bibles/deities
+	if (bible_names.len)
+		bible_name = pick(bible_names)
+	if (deity_names.len)
+		deity_name = pick(deity_names)
+
+/datum/religion/proc/misc(var/mob/living/carbon/human/H, var/obj/item/weapon/storage/bible/B)
+	return TRUE // Nothing to see here, but redefined in some other religions !
+
+// The list of all religions spacemen have designed, so far.
+/datum/religion/catholic
+	name = "Catholicism"
+	deity_name = "Jesus Christ"
+	bible_name = "The Holy Bible"
+	male_adept = "Bishop"
+	female_adept = "Bishop"
+	keys = list("catholic", "catholicism", "roman catholicism")
+
+/datum/religion/catholic/misc(var/mob/living/carbon/human/H)
+	H.equip_or_collect(new /obj/item/clothing/head/mitre(H), slot_head)
+
+/datum/religion/theism
+	name = "Theism"
+	deity_name = "God"
+	bible_names = list("The Gnostic Bible", "The Dead Seas Scrolls")
+	keys = list("theist", "gnosticism", "theism")
+
+/datum/religion/satanism
+	name = "Satanism"
+	deity_name = "Satan"
+	bible_name = "The Satanic Bible" //What I found on Google, ergo the truth
+	male_adept = "Magister"
+	female_adept = "Magistera"
+	keys = list("satan", "evil", "satanism")
+
+/datum/religion/lovecraft
+	name = "Esoteric order of Dagon"
+	deity_name = "Cthulhu" //I hope it's spelt correctly
+	bible_names = list("The Necronomicon", "The Book of Eibon", "De Vermis Mysteriis", "Unaussprechlichen Kulten")
+	keys = list("cthulhu", "old ones", "great old ones", "outer gods", "elder gods", "esoteric order of dagon")
+
+/datum/religion/islam
+	name = "Islam"
+	deity_name = "Allah"
+	bible_name = "The Quran"
+	male_adept = "Imam"
+	female_adept = "Imam"
+	keys = list("islam", "muslim")
+
+/datum/religion/slam
+	name = "Slam"
+	deity_name = "Charles Barkley"
+	bible_name = "Barkley: Shut Up and Jam - Gaiden"
+	male_adept = "Master of Slam"
+	female_adept = "Mistress of Slam"
+	keys = list("slam", "bball", "basketball", "basket ball")
+
+/datum/religion/judaism
+	name = "Judaism"
+	deity_name = "Yahweh"
+	bible_names = list("The Torah", "The Talmud")
+	male_adept = "Rabbi"
+	female_adept = "Rabbi"
+	keys = list("jew", "judaism", "jews")
+
+/datum/religion/hinduism
+	name = "Hinduism"
+	deity_names = list("Brahma", "Vishnu", "Shiva", "Ganesha")
+	bible_names = list("The Vedas", "The Mahabharata")
+	male_adept = "Guru"
+	female_adept = "Guru"
+	keys = list("hindu", "hinduism", "india")
+
+/datum/religion/buddism
+	name = "Buddism"
+	deity_name = "Buddha"
+	bible_name = "The Tripitaka"
+	male_adept = "Guru"
+	female_adept = "Guru"
+	keys = list("hindu", "hinduism", "india")
+
+/datum/religion/shintoism
+	name = "Shintoism"
+	deity_name = "Kami" //Polytheist and shit, do I sound like a weeb ?
```

Comment:
Weaboo memes aside, gods like Izanagi, Izanami, Susanoo, Amaterasu and Tsukuyomi would be appropriate